Snippets/Metrics/Metrics_1/Configuration.cs

```diff
@@ -44,6 +44,28 @@ public void ConfigureEndpoint(EndpointConfiguration endpointConfiguration)
                 interval: TimeSpan.FromSeconds(5));
 
             #endregion
+
+            #region Metrics-Observers [1.1,)
```

Comment:
@Particular/docs-maintainers How can I mark this snippet to get it properly displayed in `metrics_reporting_metrics_[1.1,).partial.md` ? Am I getting this versioning wrong?